jwst/coron/imageregistration.py

```diff
@@ -148,20 +147,21 @@ def align_array(reference, target, mask=None, return_aligned=True):
         A 2D (``NxK``) image indicating pixels to ignore when performing the
         minimization. The masks acts as a weighting function in performing
         the fit.
+    return_aligned : bool, default=True
```

Comment:
No need to specify the default - it's in the function signature.

```suggestion
    return_aligned : bool
```